drivers/can/can.c

```diff
@@ -1280,7 +1280,10 @@ int can_receive(FAR struct can_dev_s *dev, FAR struct can_hdr_s *hdr,
                  sizeof(struct can_hdr_s));
 
           nbytes = can_dlc2bytes(hdr->ch_dlc);
-          memcpy(fifo->rx_buffer[fifo->rx_tail].cm_data, data, nbytes);
+          if (nbytes)
```

Comment:
why check nbytes ? memcpy can handle size == 0

Reply:
Yes, I will delete this condition